pkg/liqonet/ipam/ipam_test.go

```diff
@@ -1559,8 +1559,82 @@ var _ = Describe("Ipam", func() {
 			})
 		})
 	})
+
+	Describe("SetReservedSubnets", func() {
+		var (
+			toBeReservedSubnets1          []string
+			toBeReservedSubnets2          []string
+			toBeReservedSubnetsIncorrect1 []string
+		)
+
+		BeforeEach(func() {
+			toBeReservedSubnets1 = []string{"192.168.0.0/16", "100.200.0.0/16", "10.200.250.0/24"}
+			toBeReservedSubnets2 = []string{"192.168.1.0/24", "100.200.0.0/16", "172.16.34.0/24"}
+			toBeReservedSubnetsIncorrect1 = []string{"192.168.0.0/16", "100.200.0/16", "10.200.250.0/24"}
+		})
+		Context("Reserving subnets", func() {
+			When("Reserving subnets for the first time", func() {
+				It("should reserve all the the subnets and return nil", func() {
+					Expect(ipam.SetReservedSubnets(toBeReservedSubnets1)).To(BeNil())
```

Comment:
```suggestion
					Expect(ipam.SetReservedSubnets(toBeReservedSubnets1)).To(Succeed())
```

Nit. Just a bit more idiomatic when asserting something succeeded.